data/assets/examples/screenspacerenderable/screenspacerenderablerenderable/renderablerenderable_model.asset

```diff
@@ -0,0 +1,47 @@
+-- Model
+-- Creates a screenspace image that renders a 3D model of the Eiffel tower into the screen
+-- space window. As the objects are rendered in meter scale, and the Eiffel tower is about
+-- 300m tall, we both shrink the rendering to make the entire model fit into the view and
+-- also modify the position of the camera
```

Comment:
There are two different ways of saying screenspace in the same sentence here, but maybe not a problem :) I'll let you decide